ci/debian-installer/preseed.cfg.tmpl

```diff
@@ -0,0 +1,507 @@
+#_preseed_V1
+#### Contents of the preconfiguration file (for bullseye)
+### Localization
+# Preseeding only locale sets language, country and locale.
+d-i debian-installer/locale string en_US
+
+# The values can also be preseeded individually for greater flexibility.
+#d-i debian-installer/language string en
+#d-i debian-installer/country string NL
+#d-i debian-installer/locale string en_GB.UTF-8
+# Optionally specify additional locales to be generated.
+#d-i localechooser/supported-locales multiselect en_US.UTF-8, nl_NL.UTF-8
+d-i localechooser/supported-locales multiselect en_US.UTF-8, \
+                                                es_ES.UTF-8, \
+                                                fr_FR.UTF-8, \
+                                                fr_CA.UTF-8, \
+                                                de_DE.UTF-8, \
+                                                he_IL.UTF-8, \
+                                                it_IT.UTF-8, \
+                                                nb_NO.UTF-8, \
+                                                nl_NL.UTF-8, \
+                                                pl_PL.UTF-8, \
+                                                pt_BR.UTF-8
+
```

Comment:
Can be removed, see https://github.com/inverse-inc/packetfence/commit/3298704a27b444eb3319debe8bec2ea987c96ff1